lib/mustache_context.dart

```diff
@@ -124,13 +124,22 @@ class _MustacheContext extends MustacheToString implements MustacheContext {
   }
 
   dynamic _getActualValue(String key) {
-    try {
-      return ctx[key];
-    } catch (NoSuchMethodError) {
-      //Try to make dart2js understand that when we define USE_MIRRORS = false
-      //we do not want to use any reflector
-      return (useMirrors && USE_MIRRORS) ? ctxReflector[key] : null;
+    if (reflect(ctx).type.instanceMembers.containsKey(new Symbol("[]"))) {
```

Comment:
This check should be done if and only if `(useMirrors && USE_MIRRORS)` otherwise the code will get transpiled to js with a lot more code, unless something changed on the dart side.